### ISSUE TITLE
can i use pixi-animate in pixijs ?

### ISSUE THREAD

**Issue by windreamlion** [pixijs-userland/animate#85]
I have had a problem for a long time. Can I use pixi-animate to load files created by animate cc in pixijs? After all, animate cc creates animations very conveniently.  thanks！

**Comment by bigtimebuddy** [comment]
Yes. You need to use the pixi-animate-extension, to export PixiJS content from Animate.

**Comment by windreamlion** [comment]
Is there a demo code for reference? thanks

**Comment by ngcamu** [comment]
Open the html page exported from Animate and you'll see the standard way to work with the assets.

`var scene = new PIXI.animate.Scene(800, 600, {
       view: document.getElementById("stage"),
       backgroundColor: 0xffffff,
       antialias: true
});
scene.load(lib.Main);`

Or are you looking for information on using a pixi-animate Scene with other display objects in a pixi-js Application?
